### ISSUE TITLE
[Package Issue]: Microsoft.Skype

### ISSUE THREAD

**Issue by phpcitizen** [microsoft/winget-pkgs#128661]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installation issue.

### Brief description of your issue

Somehow the installed version shows 8.109 and the available 8.109.0.209. 

### Steps to reproduce

winget install --uninstall-previous Microsoft.Skype

### Actual behavior

Version shows 8.109

### Expected behavior

Version should show 8.109.0.209

### Environment

```raw
Windows Package Manager v1.6.3133                                                                                       Copyright (c) Microsoft Corporation. All rights reserved.                                                                                                                                                                                       Windows: Windows.Desktop v10.0.22631.2715                                                                               System Architecture: X64                                                                                                Package: Microsoft.DesktopAppInstaller v1.21.3133.0                                                                                                                                                                                             Winget Directories                                                                                                      ----------------------------------------------------------------------------------------------------------------------- Logs                               %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\Diag… User Settings                      %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\sett… Portable Links Directory (User)    %LOCALAPPDATA%\Microsoft\WinGet\Links                                                Portable Links Directory (Machine) C:\Program Files\WinGet\Links                                                        Portable Package Root (User)       %LOCALAPPDATA%\Microsoft\WinGet\Packages                                             Portable Package Root              C:\Program Files\WinGet\Packages                                                     Portable Package Root (x86)        C:\Program Files (x86)\WinGet\Packages                                               Installer Downloads                %USERPROFILE%\Downloads                                                                                                                                                                                      Links                                                                                                                   ---------------------------------------------------------------------------                                             Privacy Statement   https://aka.ms/winget-privacy                                                                       License Agreement   https://aka.ms/winget-license                                                                       Third Party Notices https://aka.ms/winget-3rdPartyNotice                                                                Homepage            https://aka.ms/winget                                                                               Windows Store Terms https://www.microsoft.com/en-us/storedocs/terms-of-sale                                                                                                                                                                     Admin Setting                             State                                                                         --------------------------------------------------                                                                      LocalManifestFiles                        Disabled                                                                      BypassCertificatePinningForMicrosoftStore Disabled                                                                      InstallerHashOverride                     Disabled                                                                      LocalArchiveMalwareScanOverride           Disabled
```

**Comment by stephengillie** [comment]
This might be due to wingetbot sometimes adding extraneous version data.

**Comment by treysis** [comment]
It's because Skype doesn't (anymore?) add the version after 8.109.* to the ARP table.

**Comment by mdanish-kh** [comment]
Close with reason: Resolved by PR #128781;
